composer.lock

```diff
@@ -5640,5 +5648,5 @@
     "prefer-lowest": false,
     "platform": [],
     "platform-dev": [],
-    "plugin-api-version": "2.3.0"
+    "plugin-api-version": "2.2.0"
```

Comment:
@jazzsequence this and the dep downgrades above seem suspect but I don't know Composer well enough so I'm raising to your attention.

Reply:
Ah, I just discovered I think this is because @rwagner00 was running PHP 7.4 locally. I just did the exact same thing as I was also using 7.4 locally for probably the same reason. When I switch back to 8.1 and run composer install, it installs successfully. 

Ryan, likely good to revert this assuming you can switch back to 8.X locally and _this project_ wasn't the reason you were running PHP7.